<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeGUI</class>
 <widget class="QWidget" name="HomeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeGUI</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>301</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Friend list</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>filters : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_filter"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_friends"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>411</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="conversation_name">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Id to call</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="id_to_call"/>
      </item>
      <item>
       <widget class="QPushButton" name="call_btn">
        <property name="text">
         <string>call</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listView_message"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_message"/>
      </item>
      <item>
       <widget class="QPushButton" name="send_btn">
        <property name="text">
         <string>send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-image: url(:/LogGUI/power-button-outline.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_statusCall">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LogGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit_filter</sender>
   <signal>textChanged(QString)</signal>
   <receiver>listWidget_friends</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
